src/main.ts

```diff
@@ -40,11 +40,13 @@ import { SettingsTab } from 'src/settingsTab';
 import { LocationSearchDialog } from 'src/locationSearchDialog';
 import { TagSuggest } from 'src/tagSuggest';
 import * as utils from 'src/utils';
+import { MapviewAPI } from './mapviewAPI';
 
 export default class MapViewPlugin extends Plugin {
     settings: PluginSettings;
     public highestVersionSeen: number = 0;
     public iconCache: IconCache;
+    public mapviewAPI: MapviewAPI;
```

Comment:
Let's bee consistent about the naming, this should be called `mapViewAPI` (capital `V`)

Reply:
👍